examples/chat/common/src/proto/chat.rs

```diff
@@ -48,33 +47,7 @@ impl CreateRoomRequest {
         ::std::default::Default::default()
     }
```

Comment:
That's a good question, and I think the answer is "no", since for real applications (e.g. for Android) the client will still be protobuf / gRPC (or JSON / REST, I guess). Which is why although I like FIDL I am still a bit hesitant to push forward with it, since we would still need to support gRPC.